sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_get_detailed_diagnostics_information_async.py

```diff
@@ -63,6 +65,9 @@ def callback(resp):
                 raw_response_hook=callback
             )
 
+        for idx, json_responses in enumerate(json_respones):
```

Comment:
you can probably just print the entire json response, it doesn't come back per-document